cfme/tests/automate/test_instance.py

```diff
@@ -10,6 +10,10 @@
 
 pytestmark = [test_requirements.automate]
 
+STATE1 = fauxfactory.gen_alpha()
```

Comment:
Are these constants necessary? What is the reason you've decided not to put them in the fixture?

Reply:
I have added these variable into the fixture and yielded